ryft-ui/src/main/java/com/ryftpay/android/ui/model/RyftCardType.kt

```diff
@@ -38,15 +44,19 @@ internal enum class RyftCardType(
             RyftCardBinRange(min = 34, max = 34),
             RyftCardBinRange(min = 37, max = 37)
         ),
-        iconDrawableId = R.drawable.ic_ryft_amex
+        iconDrawableId = R.drawable.ic_ryft_amex,
+        availableForGooglePay = false,
+        googlePayName = "AMEX"
     ),
     Unknown(
         displayName = "Unknown",
         cardLengths = intArrayOf(19),
         cvcLength = 4,
         cardNumberFormatGaps = intArrayOf(),
         binRanges = listOf(),
-        iconDrawableId = R.drawable.ic_ryft_unknown_card
+        iconDrawableId = R.drawable.ic_ryft_unknown_card,
+        availableForGooglePay = false,
+        googlePayName = "INVALID"
```

Comment:
worth making `googlePayName` nullable and removing the `availableForGooglePay` (or keeping it but updating the `getGooglePaySupportedTypeNames` to handle null)?

Reply:
yeah good shout 👍 updated now, I've kept `availableForGooglePay` for now as it's abit clearer in the Amex case